VERSIONING.md

```diff
@@ -0,0 +1,9 @@
+# Versioning
```

Comment:
Except of the semver part, I find this a bit messy.
Is there an example of such an instruction set on other projects? It will be nice to see it a bit more structural and less like a story. I guess the main goal is to define and understand what to expect in each version, right?